packages/api/src/schema/api-schema.yaml

```diff
@@ -1875,7 +2035,43 @@ paths:
               schema:
                 $ref: "#/components/schemas/error"
     get:
-      description: Lists streams
+      tags:
+        - Stream
+      summary: Retrieve streams
+      description:
+        "GET /stream to retrieve all stream objects with the same userId. The
+        response will include both parent stream objects and the historic stream
+        objects with a parentId (stream objects representing a single live
+        stream session). This historic type of \"child-session\" stream object
+        will be deprecated in an upcoming version of the API in favor of the
+        session object.\n\nGET /stream?streamsonly=1 to retrieve all stream
+        objects with the same userId and exclude the historic stream objects
+        with a parentId (stream objects representing a single live stream
+        session).\n\n ### Quick filters\n\n|
+        Filter\t                                      |
+        Example\t                                      \t                      |
+        Description                             |\n|
+        --------------------------------------------- |
+        --------------------------------------------------------------------- |
+        -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|\n|
+        All stream objects with the same userId\t      |
+        /api/stream?streamsonly=1\t     \t     \t     \t     \t     \t     \t     \t  |
+        excludes the historic stream objects with a parentId (stream objects
+        representing a single live stream
+        session)                                                                                                                                                          |\n|
```

Comment:
I can't really get what's happening here 😆 
Is this like the markdown form old docs injected here? Does redocly support this? Could you inline the string in another YAML format so it's also readable here? The "create a webhook" one is much nicer

Reply:
Yes, it is a table :D

I know it is confusing, but yes the openapi support markdown